functions/src/utils/agora.js

```diff
@@ -0,0 +1,41 @@
+const functions = require("firebase-functions");
+
+const { RtcTokenBuilder, RtcRole } = require("agora-access-token");
+
+const AGORA_CONFIG = functions.config().agora;
+
+const appId = AGORA_CONFIG.app_id;
+const appCertificate = AGORA_CONFIG.app_certificate;
+
+// @debt should we move this into AGORA_CONFIG, or maybe venue config in firestore or similar?
+const expirationTimeInSeconds = 3600;
```

Comment:
IMHO the `AGORA_CONFIG` should contain only Agora console-related stuff. And `expirationTimeInSeconds` is used for our own purposes. It's fine to set it in `src/settings.ts`

Reply:
Backend functions don’t have access to src/settings